src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -189,9 +192,9 @@ impl Renderer for HtmlHandlebars {
         book.write_file("_FontAwesome/fonts/FontAwesome.ttf", theme::FONT_AWESOME_TTF)?;
 
         // Ace is a very large dependency, so only load it when requested
-        if book.uses_ace() {
+        if playpen_config.is_editable() {
             // Load the editor
-            let editor = editor::Editor::new(book.get_ace_path());
+            let editor = editor::Editor::new(playpen_config.get_editor());
             book.write_file("editor.js", &editor.js)?;
             book.write_file("ace.js", &editor.ace_js)?;
```

Comment:
The only gripe I have with this change left is that "editor" everywhere is not really that informative (both in config and through out the code). Maybe something along the lines of `js_editor_config` ?